### ISSUE TITLE
feat(YouTube - Swipe controls): Remember Current Brightness At ALL Times

### ISSUE THREAD

**Issue by sliceanddice9** [ReVanced/revanced-patches#2758]
### Application

YouTube ReVanced Version 18.23.35

### Issue

When changing the brightness number for a video, closing the app and then opening the app back up again - the brightness number for any video you watch resets to the default auto value of 0.

### Patch

When setting the brightness for videos in landscape/horizontal fullscreen mode via the swipe control brightness gesture, have it stick to whatever brightness number you last configured it to instead of the brightness number defaulting to 0 every time you close the app.

### Motivation

YouTube Vanced & YouTube ReVanced Extended via the swipe control brightness gesture, locks the brightness number for all videos you previously set it to even when closing the app followed by opening the app back up. Would love to have that same functionality for YouTube ReVanced as well. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by Fred-Vatin** [comment]
It also needs to remember brightness for any new video played in a playlist. Currently, it is reset when next video starts.

**Comment by oSumAtrIX** [comment]
Swipe controls can do that by enabling to remember the brightness value in settings

**Comment by sliceanddice9** [comment]
> Swipe controls can do that by enabling to remember the brightness value in settings

Wrong. Did you read the OP? The brightness value resets to the default brightness value of 0 AFTER YOU FORCE CLOSE REVANCED AND THEN RE-OPEN REVANCED.
